Lombiq.Gulp.Extensions.csproj

```diff
@@ -7,10 +7,11 @@
 
   <ItemGroup>
     <None Remove="node_modules\**" />
+    <FilesToMove Include="$(ProjectDir)\node_modules\**"/>
```

Comment:
Both `ProjectDir` and `SolutionDir` include a trailing slash (see https://docs.microsoft.com/en-us/cpp/build/reference/common-macros-for-build-commands-and-properties?view=msvc-160) so they shouldn't be followed by one.